### PR TITLE
initial commit for java sample code

### PR DESCRIPTION
**Pull request by jayvala47**

### CODE REVIEW

File: allocations/src/main/java/com/github/chrishantha/sample/allocations/AllocationsApplication.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.chrishantha.sample.allocations;
+
+import com.beust.jcommander.Parameter;
+import com.github.chrishantha.sample.base.SampleApplication;
+
```
Comment (narendratripathi) [inline allocations/src/main/java/com/github/chrishantha/sample/allocations/AllocationsApplication.java:20]: Why r w using external libraries

Comment (clbanda) [reply allocations/src/main/java/com/github/chrishantha/sample/allocations/AllocationsApplication.java:20]: its okay Narendra

**Review by clbanda — APPROVED** [review]

**Review by gauravf16 — APPROVED** [review]
OK

**Review by sujitkumar23 — COMMENTED** [review]
write the code in short

File: allocations/src/main/java/com/github/chrishantha/sample/allocations/AllocationsApplication.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 M. Isuru Tharanga Chrishantha Perera
```
Comment (barcindia) [inline allocations/src/main/java/com/github/chrishantha/sample/allocations/AllocationsApplication.java:2]: Do not copy Code

**Comment by SHRIKANTHMULUKA** [comment]
Too long in one go

**Review by rohit062019 — COMMENTED** [review]
Please get it done..

**Review by PranaliAD — APPROVED** [review]
Description of Contributor

**Comment by RajeshKumar981** [comment]
Go a head

**Comment by MilindMarathe** [comment]
OK

**Comment by NiteshBhu** [comment]
Let we check

**Comment by 2907-ashish** [comment]
OK

**Comment by SHRIKANTHMULUKA** [comment]
let merge

**Comment by RajeshKumar981** [comment]
ok

**Pull request merged**